### ISSUE TITLE
SgCore

### ISSUE THREAD

**Issue by PhuongK589** [PhuongK589/ForAitalMac#1]
Hi
I am Phuong, working c++ sgcore simulation app.

**Comment by grandixximo** [comment]
https://github.com/user-attachments/assets/a51ea8d6-46b0-49df-9894-6e16570424bf

**Comment by grandixximo** [comment]
https://github.com/user-attachments/assets/412a74fc-a5f0-4f82-9429-e174acf30eff


https://github.com/user-attachments/assets/0a7608a4-006f-4e90-8705-4aaf69c30b05

**Comment by grandixximo** [comment]
https://github.com/user-attachments/assets/059389f3-d2df-418d-93d8-9dadc5f026c6

**Comment by grandixximo** [comment]
https://github.com/user-attachments/assets/75f898a7-052e-4bb3-9564-5ab8198e13a0

**Comment by PhuongK589** [comment]
Hi

**Comment by PhuongK589** [comment]
I will check yours carefully and let u know

**Comment by PhuongK589** [comment]
Hi, I understand your issues.
now My skype has some issue. it will solve soon.
I will fix the code and send.

**Comment by grandixximo** [comment]
Can you move over to slack?

**Comment by PhuongK589** [comment]
did u invite slack?

**Comment by PhuongK589** [comment]
i will send the code soon when i fix your matters.

**Comment by grandixximo** [comment]
Yes, I did add you into out channel, but it's telling me you never logged into slack

**Comment by grandixximo** [comment]
It told me it would send you an email, check your google email

**Comment by PhuongK589** [comment]
i will

**Comment by PhuongK589** [comment]
maybe my server for skype and slack is not working well now

**Comment by PhuongK589** [comment]
it will be solved tomorrow.
i will send the screenshot of updated now

**Comment by PhuongK589** [comment]
![Screenshot_1](https://github.com/user-attachments/assets/4b399a1a-28b0-4236-b41d-9d2432683bd3)

**Comment by PhuongK589** [comment]
i will send the updated app.cpp file.
plz, replace with it and run

**Comment by PhuongK589** [comment]
[App.zip](https://github.com/user-attachments/files/17455205/App.zip)

**Comment by PhuongK589** [comment]
i sent via zip because i can't sent cpp file here.
plz, unzip and check the App.cpp file.

**Comment by grandixximo** [comment]
![image](https://github.com/user-attachments/assets/9746b269-7033-4cdc-a58d-8797347d9a43)
show me top view, doesn't look right, are you looking at it from the bottom?
![image](https://github.com/user-attachments/assets/09cf1dd2-de85-410e-bb63-40c350c3dd4b)
Like this?

**Comment by PhuongK589** [comment]
yes

**Comment by PhuongK589** [comment]
how can u get this image?

**Comment by PhuongK589** [comment]
did u run the code with updated code?

**Comment by grandixximo** [comment]
I am using AutoCAD, it is just a drawing tool, I am drawing what it should look like according to path and rotation, it can be done with any geometrical drawing tool that lets you define coordinates sizes angle and so on.

**Comment by grandixximo** [comment]
https://github.com/user-attachments/assets/55dc6074-31bf-4993-9be7-944a9f992d55

**Comment by grandixximo** [comment]
https://github.com/user-attachments/assets/769f51a9-3166-468e-98d2-50b2ccac069d

**Comment by grandixximo** [comment]
https://github.com/user-attachments/assets/42bfc2fd-712f-49c0-a883-bbf1e16e33bf

**Comment by PhuongK589** [comment]
![Screenshot_1](https://github.com/user-attachments/assets/0e0be5ad-e557-4747-b7a3-3234e72e3b62)

**Comment by PhuongK589** [comment]
i got this when C1=270 and C2=360

**Comment by grandixximo** [comment]
Are you sure you send me correct code? it does not work like shown in your piccure when we run it

**Comment by PhuongK589** [comment]
i will send again

**Comment by PhuongK589** [comment]
[App.zip](https://github.com/user-attachments/files/17455433/App.zip)

**Comment by PhuongK589** [comment]
this is the updated cpp file.

**Comment by grandixximo** [comment]
We have update, and it's the same, did you change other files?

**Comment by PhuongK589** [comment]
i didn't

**Comment by grandixximo** [comment]
YangYang built your code, and we do not get the same result, there is no way that we have found where it does what you are showing.

**Comment by grandixximo** [comment]
Did you see the video?

**Comment by PhuongK589** [comment]
i did

**Comment by PhuongK589** [comment]
i will check the code and let u know again

**Comment by grandixximo** [comment]
We can't share code here, it's pubblic

**Comment by PhuongK589** [comment]
oh, then chat via private way

**Comment by PhuongK589** [comment]
https://github.com/PhuongK589/cppsgcore/issues/1
i created private chat

**Comment by grandixximo** [comment]
But your slack and your skype don't work well, they keep breaking, if you don't resolve the communication issue permanently, we cannot continue.

**Comment by PhuongK589** [comment]
i will solve it tomorrow

**Comment by grandixximo** [comment]
I cannot view it it's private, if it's not solved tomorrow, we will put a stop to the project.

**Comment by PhuongK589** [comment]
hi, one sec plz

**Comment by PhuongK589** [comment]
https://github.com/PhuongK589/cppsgcore/issues/1
plz, check again

**Comment by PhuongK589** [comment]
https://github.com/PhuongK589/cppsgcore/issues/1
